framework/src/org/checkerframework/framework/flow/CFAbstractStore.java

```diff
@@ -809,7 +820,13 @@ public S leastUpperBound(S other) {
         {
             V otherVal = other.thisValue;
             V myVal = thisValue;
-            V mergedVal = myVal == null ? null : myVal.leastUpperBound(otherVal);
+            V mergedVal =
```

Comment:
This logic is again duplicated multiple times in this file.
Why is this the only time that the value can be null?